src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -98,9 +103,9 @@ protocol SequenceAnnotations {
     }> attributes;
 
     /**
-    Identifiers of children nodes
+    Identifier for the parent annotation, if any.
     */
-    array<string> childrenIds;
+    string parentId;
```

Comment:
Most of the comments still refer to 'annotations' where they should probably say features. All of the URLs should probably be renamed for the methods. `annotationSetId` should be `featureSetId`. Filenames should possibly changed from the sequenceAnnotations prefix. Methods still all say SeqAnn.